src/device/router_device_worker.erl

```diff
@@ -1651,6 +1660,8 @@ handle_frame_timeout(
                 },
                 Device0
             ),
+            %% Handle sequential changes to rx_delay: prev was ack'd, now a potentially new RxDelay
+            RxDelay = maps:get(rx_delay, router_device:metadata(Device0), 0),
```

Comment:
Pulling `RxDelay` all the way out of the `case` causes the next line to expect the binding to match.

Reply:
Good catch.  I must have messed that up when resolving the earlier conflict from merging `master` into this branch.